test/index.test.js

```diff
@@ -89,22 +89,21 @@ describe('KISSmetrics', function() {
         window.KM_SKIP_PAGE_VIEW = 1;
       });
 
-      it('should record normal kissmetrics page views when the option is set', function() {
-        window.KM_SKIP_PAGE_VIEW = false;
-        analytics.page();
-        analytics.didNotCall(window._kmq.push);
-      });
-
       it('should call `KM.pageView()` when KM_SKIP_PAGE_VIEW is not set', function() {
         window.KM_SKIP_PAGE_VIEW = false;
         analytics.page();
-        analytics.calledOnce(window.KM.pageView);
+        analytics.called(window._kmq.push, ['record', 'Page View', {
+          'Viewed URL': window.location.href,
+          Referrer: document.referrer || 'Direct'
```

Comment:
Is there a way to mock `document.referrer` here? It's empty in this particular test.